Arcemi.Pathfinder.SaveGameEditor/Pages/About.razor

```diff
@@ -24,14 +24,6 @@
     <ExternalLink Target="ExternalLinks.WikiFeatCodes"/>
 </p>
 <h2>Tricks</h2>
-<p>
```

Comment:
I removed this while developing the class downgrade stuff since it's no longer accurate, but since it's dummied out maybe it should come back? I don't think people should rely on it going forward considering feats can be added pretty easily now.

Reply:
I agree, and I think we can leave it as removed, it's not that useful information anymore with feats capability